packages/router-plugin/src/core/code-splitter/compilers.ts

```diff
@@ -319,23 +319,23 @@ export function compileCodeSplitVirtualRoute(opts: ParseAstOptions) {
             splitNode = binding?.path.node
           }
 
+          let componentId = 'SplitComponent'
+
           // Add the node to the program
           if (splitNode) {
             if (t.isFunctionDeclaration(splitNode)) {
+              componentId = splitNode.id?.name || componentId
               programPath.pushContainer(
```

Comment:
This change is great. Now the component is registered to fast refresh. Ideally people should not write anonymous component, but in that case giving it the name `SplitComponent` is better than having anonymous one